packages/firestore/src/api/timestamp.ts

```diff
@@ -92,4 +94,25 @@ export class Timestamp {
       ')'
     );
   }
+
+  /**
```

Comment:
How about:

```
/**
   * Converts this object to a primitive string, which allows Timestamp objects to be compared
   * using the `>`, `<=, `>=` and `>` operators.
   */
```

Rationale:
- You are providing the override. A user doesn't need to do further overriding.
- We generally don't link to GitHub issues in top-level method comments. Feel free to move the link to the implementation if you think it provides value.
- We are in JavaScript land, so I dropped the explicit mention of the language name.

You also need to copy this API to https://github.com/firebase/firebase-js-sdk/blob/master/packages/firebase/index.d.ts and https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore-types/index.d.ts

Reply:
Done